bedrock/firefox/templates/firefox/fights-for-you.de.html

```diff
@@ -1,146 +0,0 @@
-{# This Source Code Form is subject to the terms of the Mozilla Public
- # License, v. 2.0. If a copy of the MPL was not distributed with this
- # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
-
-{% from "macros.html" import google_play_button with context %}
-{% from "macros-protocol.html" import call_out, call_out_compact, hero, feature_card with context %}
-
-{% extends "firefox/base/base-protocol.html" %}
-
-{% set referrals = '?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=fffy-page' %}
-
-{% block page_title %}Firefox Fights for you. Wir schützen Deine Daten und verkaufen sie niemals.{% endblock %}
-{% block page_desc %}Firefox respektiert Deine Privatsphäre und macht es Dir leicht, Deine Daten zu schützen. Mit klaren Privatsphäre-Optionen und Tracking-Schutz. Du entscheidest, was Du teilst und wann.{% endblock %}
-
-{% block page_css %}
-  {{ css_bundle('fights-for-you') }}
-{% endblock %}
-
-{% block extrahead %}
-<!--[if lt IE 10]>
-  <style>
-    /* avoid IE9 background calc() bug */
-    .fffy-c-hero,
-    .mzp-c-call-out-compact.fffy-c-footer-callout {
-    background: #2A0140 !important;
-  }
-  </style>
-<![endif]-->
-{% endblock %}
-
-{% block body_id %}fights-for-you{% endblock %}
-
-{% block content %}
-
-<div class="fffy-c-hero">
-  {% call hero(
-    title='Firefox Fights For You',
-    desc='Privatsphäre sollte einfach sein. Daten sicher. Einstellungen eindeutig. Wir kämpfen für ein besseres Internet – mit Produkten, die Dir auch online die Kontrolle über Dein Leben geben.',
-    class='mzp-has-image mzp-t-firefox mzp-t-product-firefox mzp-t-dark',
-    include_cta=True
-  ) %}
-  <p><a id="fffy-video-button" href="https://www.youtube.com/watch?v=3NG6dZl9kpA" data-id="3NG6dZl9kpA" class="mzp-c-button mzp-t-secondary mzp-t-dark mzp-t-small" data-link-type="button" data-link-name="Watch video">Video anschauen</a></p>
-  {% endcall %}
-
-  <div class="fffy-c-product mzp-l-content">
-    {{ high_res_img('firefox/fights-for-you/hero.png', {'alt': '', 'width': '904', 'height': '646'}) }}
```

Comment:
Looks like there are images in the `/media/img/firefox/fights-for-you/` directory that can be deleted.